README.md

```diff
@@ -1 +1,55 @@
-# irida-metadata-uploader
\ No newline at end of file
+# IRIDA Metadata Uploader
+
+Python program for uploading metadata to [IRIDA](https://github.com/phac-nml/irida)
+
+For NGS data uploads, see [IRIDA Uploader](https://github.com/phac-nml/irida-uploader/)
+
+This projects code has been adapted from the much larger [ARTIC Nextflow pipeline project](https://github.com/phac-nml/ncov2019-artic-nf)
+
+## Run from source code
+
+```bash
+make
+source .virtualenv/bin/activate
+irida-metadata-uploader --help
+```
+
+### Running the metadata uploader
+
+Usage
+```bash
+usage: irida-metadata-uploader [-h] -c CONFIG -m METADATA_CSV [--no_sample_creation]
+```
+
+#### CONFIG
+
+Specify a config file for connection to IRIDA. This is identical to an IRIDA Uploader config file.
+
+[Click here for more infomation about this file](https://irida-uploader.readthedocs.io/en/stable/configuration.html)
+
+#### METADATA_CSV
+
+Table must be in CSV format and must contain `sample` and `project_id` as the first 2 columns. All other columns will be processed as metadata.
```

Comment:
These two columns could be found anywhere in the metadata file, just need to match the header strings of `sample` and `project_id`. I'd also mention that the project id has to be an integer value or the sample will be skipped over

Reply:
Fixed in https://github.com/phac-nml/irida-metadata-uploader/pull/1/commits/c3e2baf188e14445c9a755235a1273664e7a9081